css/CSS2/generated-content/content-100.xht

```diff
@@ -5,6 +5,7 @@
         <link rel="author" title="Microsoft" href="http://www.microsoft.com/" />
         <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#propdef-content" />
         <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#content" />
+        <link rel="match" href="content-100-ref.html" />
```

Comment:
This also fails in Firefox, FWIW, though I think this is fine?

Reply:
The attribute was depreciated in HTML5, so implementations might have slipped with some of these  (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/area).

Reply:
That shouldn't have any affect on how it goes in the DOM, though, so it's somewhat surprising.

I presume it's just something weird with the implementation of map/area, rather than the attribute itself.